crates/ruff_python_formatter/src/expression/mod.rs

```diff
@@ -115,7 +115,11 @@ impl FormatRule<Expr, PyFormatContext<'_>> for FormatExpr {
         };
 
         if parenthesize {
-            parenthesized("(", &format_expr, ")").fmt(f)
+            let comments = f.context().comments().clone();
+            let open_parenthesis_comments = comments.open_parenthesis_comments(expression);
+            parenthesized("(", &format_expr, ")")
+                .with_dangling_comments(open_parenthesis_comments)
```

Comment:
Nit: For a separate PR. Should we rename this method to `with_trailing_open_parentheses_comments`?